src/tree/images/ImageTreeItem.ts

```diff
@@ -71,6 +71,11 @@ export class ImageTreeItem extends AzExtTreeItemIntermediate {
         if (ref.endsWith(':<none>') && this._item.RepoDigests?.length) {
```

Comment:
Technically the comment in the line above is no longer true (dangling images are shown, depending on the setting), please update it. 

I am also wondering if it makes sense to simplify this code and not use the image digest at all, but switch straight to image ID if the image tag ends with `<none>`

Reply:
Yes, the comment need to be changed. I will fix it and remove the ref by digest.